devtools-panels/background_scripts/background.js

```diff
@@ -0,0 +1,19 @@
+
+/**
+When we receive the message, execute the given script in the given
+tab.
+*/
+function handleMessage(request, sender, sendResponse) {
```

Comment:
`sendResponse` is not used and can be omitted (`sender` also, but see the next comment).